app/controllers/account/users/posts_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Account::Users::PostsController < Account::PostsController
+
+  def show
+    @post = resource
+    @posts = @post.user.posts.ordered
+    respond_to do |format|
+      format.js { render 'account/users/posts/update_gallery' }
+    end
+  end
+
+  def destroy
+    @post = resource
+    @post_for_show = @post.next ? @post.next : @post.prev
+    @posts = collection.page(params[:page]).per(24)
+    @post.destroy
+    respond_to do |format|
+      format.js { render 'account/posts/destroy' }
+    end
+  end
+
+  private
+
+
+  def post_update_params
+    params.require(:post).permit(:content)
+  end
+
+  def collection
+    @user = User.find(params[:id])
+    @posts = @user.posts.ordered
```

Comment:
why do you set to a variable here?

Reply:
because i need a collection for displaying 'next-prev' posts now